doc/modules/outlier_detection.rst

```diff
@@ -246,6 +246,9 @@ This strategy is illustrated below.
   (:class:`covariance.MinCovDet`) of location and covariance to
   assess the degree of outlyingness of an observation.
 
+* See :ref:`sphx_glr_auto_examples_applications_plot_outlier_detection_wine.py` for an example of robust covariance estimation
+on a real data set.
```

Comment:
Yes, for instance,

```
this is a very long line foo bar boo bar foo bar the rest of the line
```

becomes

```
this is a very long line foo bar boo bar foo bar the
rest of the line
```

In this particular case:

```suggestion
* See :ref:`sphx_glr_auto_examples_applications_plot_outlier_detection_wine.py`
for an example of robust covariance estimation on a real data set.
```

Reply:
Ah I see, thank you very much!